GUI/SearchTaskWindow.xaml.cs

```diff
@@ -729,7 +729,26 @@ private void NonSpecificUsingNonSpecific(object sender, RoutedEventArgs e)
             }
         }
 
-        private void NonSpecificUpdate(object sender, TextChangedEventArgs e)
+        //this one is used by the GUI
+        private void ProteaseSpecificUpdate(object sender, SelectionChangedEventArgs e)
+        {
+            const int maxLength = 25;
+            if (((Protease)ProteaseComboBox.SelectedItem).Name.Contains("non-specific"))
+            {
+                MaxPeptideLengthTextBox.Text = maxLength.ToString();
+            }
+            else if (((Protease)ProteaseComboBox.SelectedItem).Name.Contains("top-down"))
+            {
+                UseProvidedPrecursor.IsChecked = false;
+                DeconvolutionMaxAssumedChargeStateTextBox.Text = "60";
```

Comment:
60 is very high in my experience doing td searches in MM. maybe 2X too high. If there are no practical consequences (e.g. longer search times, larger search spaces, more false positives), then no big deal. But i suspect there is a cost and that this number should be lower

Reply:
60 was pulled from the wiki's recommendations: https://github.com/smith-chem-wisc/MetaMorpheus/wiki/Top-Down

I have no opinion, but we should be consistent in what we recommend.

Reply:
LVS has seen high MW proteoforms up to c.s. of 50. So maybe 60 is fine.